charts/jaeger/templates/_helpers.tpl

```diff
@@ -719,3 +719,95 @@ Create pull secrets for hotrod image
 {{- define "hotrod.imagePullSecrets" -}}
 {{- include "common.images.renderPullSecrets" (dict "images" (list .Values.hotrod.image) "context" $) -}}
 {{- end }}
+
+
+
+{{- define "jaeger-v2.name" -}}
```

Comment:
we already discussed this - all these settings are already present in the chart without the `v2` name, so please use the existing ones.  I don't think there should be any properties / templates that have `v2` in the name